tests/swap.rs

```diff
@@ -832,7 +832,7 @@ async fn run_restore_checkpoint_bob_pre_buy_alice_pre_buy(
     drop(monero_wallet_lock);
     drop(lock);
     let delta_balance = after_balance.balance - before_balance.balance;
-    assert!(delta_balance > 999660000000);
+    assert!(delta_balance.as_pico() > 999660000000);
```

Comment:
NIt: I'd prefer doing the conversion on the other side, so we don't change the data or type to be tested.